docs/source/en/model_doc/hubert.md

```diff
@@ -14,88 +14,158 @@ rendered properly in your Markdown viewer.
+### Installation
 
-Next, [install](https://github.com/Dao-AILab/flash-attention#installation-and-features) the latest version of Flash Attention 2:
+Check to see if your hardware supports Flash Attention 2 by reviewing the [official compatibility list](https://github.com/Dao-AILab/flash-attention#installation-and-features). 
+If your hardware is compatible, install the latest version with:
 
 ```bash
 pip install -U flash-attn --no-build-isolation
 ```
 
+If your hardware is not compatible, you can still use attention kernel optimizations through [Better Transformer](https://huggingface.co/docs/transformers/main/en/model_doc/bark#using-better-transformer).
+
 ### Usage
 
-Below is an expected speedup diagram comparing the pure inference time between the native implementation in transformers of `facebook/hubert-large-ls960-ft`, the flash-attention-2 and the sdpa (scale-dot-product-attention) version. We show the average speedup obtained on the `librispeech_asr` `clean` validation split: 
+You can enable Flash Attention 2 by setting the attn_implementation argument to "flash_attention_2" when loading the model:
 
 ```python
->>> from transformers import HubertModel
->>> import torch
-
->>> model = HubertModel.from_pretrained("facebook/hubert-large-ls960-ft", torch_dtype=torch.float16, attn_implementation="flash_attention_2").to("cuda")
-...
+from transformers import HubertModel
+import torch
+
+model = HubertModel.from_pretrained(
+    "facebook/hubert-large-ls960-ft",
+    torch_dtype=torch.float16,
+    attn_implementation="flash_attention_2"
+).to("cuda")
 ```
 
-### Expected speedups
+> [!NOTE]
+> Flash Attention 2 currently only works with PyTorch models and requires CUDA-compatible hardware with Ampere or newer GPUs.
 
-Below is an expected speedup diagram comparing the pure inference time between the native implementation in transformers of the `facebook/hubert-large-ls960-ft` model and the flash-attention-2 and sdpa (scale-dot-product-attention) versions. . We show the average speedup obtained on the `librispeech_asr` `clean` validation split: 
+## Quantization
 
+Quantization reduces the memory burden of large models by representing the weights in a lower precision. 
+Refer to the [Quantization](https://huggingface.co/docs/transformers/en/quantization/overview) overview for more available quantization backends.
 
-<div style="text-align: center">
-<img src="https://huggingface.co/datasets/kamilakesbi/transformers_image_doc/resolve/main/data/Hubert_speedup.png">
-</div>
+The example below uses [PyTorch Dynamic Quantization](https://pytorch.org/docs/stable/quantization.html#dynamic-quantization) to only quantize the weights to 8-bit integers (int8).
```

Comment:
Let's use one of the backends in Transformers, for example bitsandbytes with the `BitsAndBytesConfig`